dashboard/src/components/flow/WorkflowEdge.test.tsx

```
@@ -0,0 +1,55 @@
+  targetX: 200,
+  targetY: 100,
+  sourcePosition: Position.Right,
+  targetPosition: Position.Left,
+  data: { label: '0:24', status: 'completed' as const },
+};
+
+const renderEdge = (overrides?: Partial<EdgeProps<WorkflowEdgeType>>) => {
+  const props = { ...baseProps, ...overrides };
+  return render(
+    <ReactFlowProvider>
+      <svg>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)